dask_image/ndfourier/__init__.py

```diff
@@ -1,12 +1,6 @@
 # -*- coding: utf-8 -*-
 
 from __future__ import division
```

Comment:
Unrelated, but we can probably drop this now that the code is Python 3 only. Raised as issue ( https://github.com/dask/dask-image/issues/226 )